planning/behavior_velocity_intersection_module/src/util.cpp

```diff
@@ -446,8 +446,18 @@ IntersectionLanelets getObjectiveLanelets(
   // get conflicting lanes on assigned lanelet
   const auto & conflicting_lanelets =
     lanelet::utils::getConflictingLanelets(routing_graph_ptr, assigned_lanelet);
-  const auto & adjacent_followings = routing_graph_ptr->following(conflicting_lanelets.back());
+  std::vector<lanelet::ConstLanelet> adjacent_followings;
 
+  if (!conflicting_lanelets.empty()) {
+    for (const auto & conflicting_lanelet : conflicting_lanelets) {
+      for (const auto & following_lanelet : routing_graph_ptr->following(conflicting_lanelet)) {
+        adjacent_followings.push_back(following_lanelet);
+      }
+      for (const auto & following_lanelet : routing_graph_ptr->previous(conflicting_lanelet)) {
+        adjacent_followings.push_back(following_lanelet);
+      }
+    }
+  }
```

Comment:
You do not need to check `conflicting_lanelets.empty()`

Reply:
@soblin I solved these